modules/js/perf/perf_imgproc/perf_sobel.js

```diff
@@ -88,44 +96,48 @@ function perf() {
         let suite = new Benchmark.Suite;
         totalCaseNum = 0;
         currentCaseId = 0;
+        let params = "";
+        let paramObjs = [];
+        paramObjs.push({name:"size", value:"", reg:[""], index:0});
+        paramObjs.push({name:"ddepth", value:"", reg:["/CV\_[0-9]+[A-z]+[A-z]+[0-9]/g"], index:1});
```

Comment:
Regexp could be a bit simpler here. First variable letter is from set `[FSUfsu]` while second is always `C`. Also generally there could be more than 9 channels, however I've never seen a test with such input.

Reply:
I have simplified it. Please review it.